lib/ansible/plugins/connection/ssh.py

```diff
@@ -352,7 +352,7 @@ def wrapped(self, *args, **kwargs):
                 if return_tuple[0] == 255:
                     SSH_ERROR = True
                     for signature in NOT_SSH_ERRORS:
-                        if signature in return_tuple[1]:
+                        if to_bytes(signature) in return_tuple[1]:
```

Comment:
Instead of doing a conversion here, just modify the NOT_SSH_ERRORS structure to contain byte strings instead:
```
 NOT_SSH_ERRORS = (b'Traceback (most recent call last):',  # Python-2.6 when there's an exception
                                                           # while invoking a script via -m
                   b'PHP Parse error:',  # Php always returns error 255
)
```